dev-tex/biblatex/biblatex-3.7-r1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit latex-package
+
+DESCRIPTION="Reimplementation of the bibliographic facilities provided by LaTeX"
+HOMEPAGE="http://www.ctan.org/tex-archive/macros/latex/contrib/biblatex https://github.com/plk/biblatex/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tds.tgz"
+
+LICENSE="LPPL-1.3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+biber doc examples"
+
+DEPEND="dev-texlive/texlive-bibtexextra
+	dev-texlive/texlive-latexextra
+	dev-texlive/texlive-genericextra"
+RDEPEND="${DEPEND}
+	biber? ( ~dev-tex/biber-2.7 )"
+
+S="${WORKDIR}"
+TEXMF=/usr/share/texmf-site
+
+src_install() {
+	insinto "${TEXMF}"
+	doins -r bibtex tex
+
+	dodoc doc/latex/biblatex/{README,CHANGES.org}
+	use doc && dodoc doc/latex/biblatex/biblatex.{pdf,tex}
```

Comment:
I think only the pdf should be installed; nobody will prefer to read the .tex :)